ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-ref.yaml

```diff
@@ -8,7 +8,8 @@ ref:
       memory: 100Mi
   env:
     - name: OS_CLOUD
-      documentation: Name of cloud to use from ${CLUSTER_PROFILE_DIR}/clouds.yaml file
+      documentation: Name of cloud to use from clouds.yaml file
+      default: "openstack"
```

Comment:
come to think of it. We don't need default here either. 
Since the variable is declared on the ref file, if it is not define somewhere along the line (in the workflow or in the chain), the make call in https://github.com/openshift/release/blob/master/Makefile#L92 will catch it. and will make sure it is defined.